### PR TITLE
Add action that builds and runs tests

### PR DESCRIPTION
**Pull request by anraf1001**

### CODE REVIEW

File: .github/actions/run_tests/run_tests.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+mkdir -p $GITHUB_WORKSPACE/.github/scripts/build
```
Comment (ziobron) [inline .github/actions/run_tests/run_tests.sh:3]: Extract a BUILD_DIR variable

File: .github/actions/run_tests/action.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  task_name:
+    description: 'Task name'
+    required: true
+  test_file:
+    description: 'Test file'
+    required: true
+  test_to_run:
+    description: 'Name of test that should be run'
+runs:
+  using: 'composite'
```
Comment (ziobron) [inline .github/actions/run_tests/action.yml:13]: What is composite?

Comment (anraf1001) [reply .github/actions/run_tests/action.yml:13]: It's a type of github action. There are javascript, docker and composite actions. Composites can only use other actions of shell scripts, so they are the most basic ones.

**Comment by ziobron** [comment]
Your PR was merged!
🏅 2 XP granted. Thanks @anraf1001 for making course materials better!

**Comment by ziobron** [comment]
Your PR was merged!
🏅 2 XP granted. Thanks @anraf1001 for making course materials better!

**Pull request merged**